core/tests/protractor_utils/workflow.js

```diff
@@ -313,16 +324,13 @@ var uploadImage = async function(
   await waitFor.visibilityOf(
     imageClickableElement,
     'Image element is taking too long to appear.');
-  await imageClickableElement.click();
+  await action.click('Image Clickable Element', imageClickableElement);
 
   if (resetExistingImage) {
-    expect(await thumbnailResetButton.isPresent()).toBe(true);
     await waitFor.elementToBeClickable(
       thumbnailResetButton,
       'Topic thumbnail reset button taking too long to appear.');
-    await thumbnailResetButton.click();
-  } else {
-    expect(await thumbnailResetButton.isPresent()).toBe(false);
+    await action.click('Thumbnail Reset Button', thumbnailResetButton);
   }
 
   absPath = path.resolve(__dirname, imgPath);
```

Comment:
Below, can you use `action.sendKeys` for `return await imageUploadInput.sendKeys(absPath);`?

Reply:
done